### PR TITLE
Vote and "urgent" box

### PR DESCRIPTION
**Pull request by EzraWeller**
Here's a shot at all these features. We will need to adjust the config once the non-dev server is serving votes on the proper narrator index we want -- something to streamline later.

Pulling and trying locally should be the best way to see the features? In a bit of a rush atm.

### CODE REVIEW

**Comment by EzraWeller** [comment]
Looks like this also includes my experiment with having the script take a previous state -- shouldn't be a problem, but an accident.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "dependencies": {}
```
Comment (jessebmiller) [inline package.json:2]: This looks like a mistake? do we need this file?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
```
Comment (jessebmiller) [inline yarn.lock:3]: This also looks like a mistake

**Review by jessebmiller — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (EzraWeller) [reply package.json:2]: Probably accidently `yarn`ed in the main repo once and missed these added files, yeah

**Pull request merged**